strapi/strapi
# fix: added the hover color as 'danger100'

What does it do?
Added the popover as 'danger100' for the background of the 'Danger' rows

Why is it needed?
To fix issue [20777](https://github.com/strapi/strapi/issues/20777): The background of “Danger” rows in a popover should be red (Danger100).

How to test it?

- Open the Content Manager.
- In the popover for any background rows labeled `Danger`, the color will appear as red `danger100` on hover.

<img width="312" alt="image" src="https://github.com/user-attachments/assets/34395af4-fc03-48c9-be4f-bf4b801cb953">


Related issue(s)/PR(s)
https://github.com/strapi/strapi/issues/20777
